README.md

```diff
@@ -72,24 +71,27 @@ Replace value of `image` field with your catalog source image.
 
 6. Deploy Che using chectl:
 ```sh
-chectl server:start --installer=olm --multiuser --platform=openshift -n che-namespace --catalog-source-yaml /home/user/path/to/custom-catalog-source.yaml --olm-channel=nightly --package-manifest-name=eclipse-che-preview-openshift
+chectl server:start --installer=olm --multiuser --platform=openshift \
+  -n che-namespace \
+  --catalog-source-yaml /home/user/path/to/custom-catalog-source.yaml \
+  --package-manifest-name=eclipse-che-preview-openshift \
+  --olm-channel=nightly 
 ```
 
 ### OpenShift oAuth
 
-Bear in mind that che-operator service account needs to have cluster admin privileges so that the operator can create oauthclient at a cluster scope.
-There is `oc adm` command in both deploy scripts. Uncomment it if you need this feature.
-Make sure your current user has cluster-admin privileges.
+Che-operator service account needs to have cluster admin privileges so that the operator can create a cluster-wide oauthclient. 
+
+Log in using a user with cluster admin privileges, then uncomment the `oc adm` commands in both deploy scripts to enable this feature.
```

Comment:
Is this section still accurate? why not just talk about the `chectl server:start --os-oauth` flag ?

Reply:
No, seems this is out of date. Due to quick evolving and number of open issues there is no time to review everything.

Reply:
so... does a user still have to do `oc adm` stuff here? 

Would it be better to replace this with:

> Log in using a user with cluster admin privileges, then run `chectl server:start ...` using the `--os-oauth` flag to enable this feature.

Reply:
I don't think user should do `oc adm`, at least for me `--os-oauth` works fine.

Reply:
cool, updated text.